app/src/main/java/org/fossasia/pslab/activity/MultimeterActivity.java

```diff
@@ -127,6 +135,11 @@ public void onState(int state) {
         reset.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
+                saveData("", "");
```

Comment:
Any specific reason to use blank strings here?

Reply:
@CloudyPadmal Through the reset button the text-box should be made blank and should remain blank thus saving the blank string is also necessary or otherwise we have to clear the shared preferences, that can also be done.

Reply:
I think you can use a key for this purpose and then use if statement to check the state of shared preference because empty string doesn't make any sense.